crates/pathfinder/src/config/cli.rs

```diff
@@ -107,6 +110,14 @@ Examples:
                 .value_name("PATH")
                 .env("PATHFINDER_DATA_DIRECTORY")
         )
+        .arg(
+            Arg::new(SEQ_URL_KEY)
+                .long(SEQ_URL_KEY)
+                .help("Sequencer REST API endpoint")
+                .long_help("Lets you customise the Sequencer address. Useful if you have a proxy in front of the Sequencer.")
+                .takes_value(true).value_name("HTTP(s) URL")
```

Comment:
```suggestion
                .takes_value(true)
                .value_name("HTTP(s) URL")
```

Interesting rustfmt, does it really want to put them on the same line?